include/constants/flags.h

```diff
@@ -1566,6 +1566,7 @@
 #define FLAG_UNUSED_0x91F                           (SYSTEM_FLAGS + 0xBF) // Unused Flag
 
 // Daily Flags
+#define DAILY_FLAGS_START                           FLAG_DAILY_0x920
 #define FLAG_DAILY_0x920                            (SYSTEM_FLAGS + 0xC0)
```

Comment:
The following daily flags should use `DAILY_FLAGS_START`, not `SYSTEM_FLAGS`, right?

Reply:
Yeah. `DAILY_FLAGS_SIZE` could also be redefined to `(FLAG_0x95F - DAILY_FLAGS_START + 1) / 8` or something along those lines